frontend/packages/kubevirt-plugin/src/components/table/validation-cell.tsx

```diff
@@ -7,12 +7,20 @@ import './validation-cell.scss';
 export type SimpleCellProps = {
   children?: React.ReactNode;
   validation?: ValidationObject;
+  additionalLabel?: string;
 };
 
-export const ValidationCell: React.FC<SimpleCellProps> = ({ children, validation }) => {
+export const ValidationCell: React.FC<SimpleCellProps> = ({
+  children,
+  validation,
+  additionalLabel,
+}) => {
   return (
     <>
-      {children}
+      <div className={additionalLabel ? 'kv__same_row' : ''}>
```

Comment:
you can use classNames here.

Reply:
Please also use the same naming conventions like the rest of the classnames in this file.

Reply:
you can convert it to kv though